lib/aws/rails/sqs_active_job/configuration.rb

```diff
@@ -81,13 +83,18 @@ class Configuration
         #   +active_job.queue_adapter = :amazon_sqs_async+.  Called with:
         #   [error, job, job_options]
         #
-        # @option options [SQS::Client] :client SQS Client to use.  A default
+        # @option options [SQS::Client] :client SQS Client to use. A default
         #   client will be created if none is provided.
+        #
+        # @option options [Array] :deduplication_keys ([])
```

Comment:
It'd be nice to convey the default here (all properties except job id) since it is not actually an empty array, or maybe the configuration should be changed to be an exclude list instead of an override (which job id can be hard coded appended to), whichever has better customer experience.

Reply:
I'd personally lean towards an exclude list instead of include list.  That way the default doesn't need to change if other fields are added in the future.  And in general, I'd imagine most use cases would want to just exclude 1 or 2 additional fields?

Reply:
Revised with feedback from alextwoods :)